pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
-    "setuptools<60.0",
+    "setuptools<68",
```

Comment:
Can you add a comment explaining "why <68?" to help people from the future to know when/if they can raise this.